lib/module.js

```diff
@@ -635,7 +635,7 @@ Module._initPaths = function() {
 
   var nodePath = process.env['NODE_PATH'];
   if (nodePath) {
-    paths = nodePath.split(path.delimiter).filter(function(path) {
+    paths = nodePath.split(path.delimiter).filter(function pathsFilterCallback(path) {
```

Comment:
I *think* this is the only one that's necessary. The rest should show up as expected in stack traces, etc. because they're attached to an object (prototype or otherwise).